shared/js/ui/models/site.es6.js

```diff
@@ -258,6 +271,36 @@ Site.prototype = window.$.extend({},
                     }
                 })
             }
+        },
+
+        generateBreakagePixel: function (category) {
```

Comment:
I think this function could be tested with some mock data for the tab, what do you think?
The cases to test would be no tab, blocked and nonblocked trackers, and then surrogates vs everything else.